### DISCUSSION TITLE
Is there a way to change the number of sub-notes shown

### DISCUSSION THREAD

**Discussion by bert010** [TriliumNext/Trilium#3599] [Q&A]
in grid /list view there's pagination when there's more then 20 notes, can this be changed somehow to e.g 50?

**Comment by zadam** [comment]
Hi, not on the UI, but you can change this via a label - adding `#pageSize=50` to a note where you want to set this.

If you want to change this globally, you can set this label on root and make the label inheritable.

**Reply by bert010** [reply]
thnkz!!
